glossary.yml

```diff
@@ -3434,6 +3434,15 @@
       Vifaa vikuu vya kompyuta yoyote ya dijitali inayojumuisha
       sakiti muhimu za kielektroniki zinazotafsiri na kutekeleza maagizo
       kutoka kwa programu au maunzi mengine.
+      
```

Comment:
@sofiiamatios If you wish now to use this to practice editing pull requests with `git rebase -i ...` this is a good candidate - remove this line above "uk" and compress all edits into one commit.

Reply:
@sofiiamatios I have handled this PR myself - since I can no merge PRs, I have used "squash and merge" strategy to keep the history of commits linear.